File: src-szk/SimDms.Web/bin/assets/js/app/wh/mp/MpDashboard.js
$(document).ready(function () {
    var options = {
        title: "Man Power Dashboard",
        xtype: "panels",
        panels: [
            {
                name: "pnlFilter",
                items: [
                    {
                        text: "Dealer Name",
                        type: "controls",
                        items: [
                            { name: "CompanyCode", text: "Dealer", cls: "span7", type: "select" },
                        ]
                    },
                    {
                        text: "Outlet Name",
                        type: "controls",
                        items: [
                            { name: "BranchCode", text: "Outlet", cls: "span7", type: "select" },
                        ]
                    },
                    {
                        text: "Periode",
                        type: "controls",
                        items: [
                            { name: "Periode", text: "Periode", cls: "span2", type: "datepicker" },
                        ]
                    },
                ],
            },
            {
                name: "InqPers",
                xtype: "k-grid",
            },
        ],
        toolbars: [
            { name: "btnRefresh", text: "Refresh", icon: "fa fa-refresh" },
            { name: "btnExportXls", text: "Export (xls)", icon: "fa fa-file-excel-o" },
        ],
    }
    var widget = new SimDms.Widget(options);
    widget.setSelect([{ name: "CompanyCode", url: "wh.api/combo/DealerList", optionalText: "-- SELECT ONE --", params: { LinkedModule: "MP" } }]);
    widget.default = { Status: "1" };
    widget.render(function () {
        renderCallback();
    });

    function renderCallback() {
        widget.populate(widget.default);

        $("[name=CompanyCode]").on("change", function () {
            widget.select({ selector: "[name=BranchCode]", url: "wh.api/combo/Branchs", params: { comp: $("[name=CompanyCode]").val() }, optionalText: "-- SELECT ALL --" });
            widget.select({ selector: "[name=Department]", url: "wh.api/combo/departments", params: { comp: $("[name=CompanyCode]").val() }, optionalText: "-- SELECT ALL --" });
            $("[name=Department]").change();
        });
        $("[name=Department]").on("change", function () {
            widget.select({ selector: "[name=Position]", url: "wh.api/combo/positions", params: { comp: $("[name=CompanyCode]").val(), dept: $("[name=Department]").val() }, optionalText: "-- SELECT ALL --" });
            $("[name=Position]").change();
        });
        $("#btnRefresh").on("click", refreshGrid);
        $("#btnExportXls").on("click", exportXls);
        $('[name="Periode"]').val(moment(new Date()).format('DD-MMM-YYYY'));
    }

    function refreshGrid() {
        if (widget.isNullOrEmpty($('[name="CompanyCode"]').val()) == false) {
            widget.kgrid({
                url: "wh.api/inquiry/MpDashboard",
                name: "InqPers",
                params: $("#pnlFilter").serializeObject(),
                //serverBinding: true,
                columns: [
                    { field: "BranchName", width: 500, title: "Outlet" },
                    { field: "BranchManager", width: 140, title: "Branch Manager", footerTemplate: '#=sum#', type: 'align-right' },
                    { field: "SalesHead", width: 140, title: "Sales Head", footerTemplate: '#=sum#', type: 'align-right' },
                    { field: "SalesCoordinator", width: 140, title: "Sales Coordinator", footerTemplate: '#=sum#', type: 'align-right' },
                    { field: "Salesman", width: 140, title: "Salesman", footerTemplate: '#=sum#', type: 'align-right' },
                    { field: "SalesmanPlatinum", width: 140, title: "Salesman Platinum", footerTemplate: '#=sum#', type: 'align-right' },
                    { field: "SalesmanGold", width: 140, title: "Salesman Gold", footerTemplate: '#=sum#', type: 'align-right' },
                    { field: "SalesmanSilver", width: 140, title: "Salesman Silver", footerTemplate: '#=sum#', type: 'align-right' },
                    { field: "SalesmanTrainee", width: 140, title: "Salesman Trainee", footerTemplate: '#=sum#', type: 'align-right' },
                    { field: "TotalSalesForce", width: 140, title: "Total Sales Force", footerTemplate: '#=sum#', type: 'align-right' },
                ],
                aggregate: [
                    { field: "BranchManager", aggregate: "sum" },
                    { field: "SalesHead", aggregate: "sum" },
                    { field: "SalesCoordinator", aggregate: "sum" },
                    { field: "Salesman", aggregate: "sum" },
                    { field: "SalesmanPlatinum", aggregate: "sum" },
                    { field: "SalesmanGold", aggregate: "sum" },
                    { field: "SalesmanSilver", aggregate: "sum" },
                    { field: "SalesmanTrainee", aggregate: "sum" },
                    { field: "TotalSalesForce", aggregate: "sum" },
                ],
            });
        }
    }

    function exportXls() {
        console.log('Exporting xls ...');
    }
});

File: src-dms/SimDms.Absence/Models/MpDashboard.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SimDms.Absence.Models
{
    public class MpDashboard
    {
        public string CompanyCode { get; set; }
        public string BranchCode { get; set; }
        public string BranchName { get; set; }
        public int? BranchManager { get; set; }
        public int? SalesHead { get; set; }
        public int? SalesCoordinator { get; set; }
        public int? Salesman { get; set; }
        public int? SalesmanPlatinum { get; set; }
        public int? SalesmanGold { get; set; }
        public int? SalesmanSilver { get; set; }
        public int? SalesmanTrainee { get; set; }
        public int? TotalSalesForce { get; set; }
    }

    public class ManPowerDashboardModel
    {
        public string BranchCode { get; set; }
        public string BranchName { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
    }

    public class SfMpDashboard
    {
        public string Outlet { get; set; }
        public Int32? BranchManager { get; set; }
        public Int32? SalesHead { get; set; }
        public Int32? Platinum { get; set; }
        public Decimal? PlatinumPct { get; set; }
        public Int32? Gold { get; set; }
        public Decimal? GoldPct { get; set; }
        public Int32? Silver { get; set; }
        public Decimal? SilverPct { get; set; }
        public Int32? Trainee { get; set; }
        public Decimal? TraineePct { get; set; }
        public Int32? TotalSalesPerson { get; set; }
        public Int32? TotalSalesForce { get; set; }
    }

}